README.adoc

```diff
@@ -69,6 +69,10 @@ A workaround was implemented to provide JGit but use Apache HTTP client for auth
 The issue in JGit has now been resolved and delivered in git client plugin releases.
 JGit with Apache HTTP Client continues to delivered to assure compatibility.
 
+[#installing-portablegit-automatically]
+=== Installing PortableGit Automatically
+The git client plugin supports installing PortableGit automatically from a zip or tar.gz file. First follow PortableGit's installation instructions and then repack so that the git executable is at `<zipname>.zip\bin\git.exe`. Leave Path to Git executable as git.exe and specify `bin\git.exe` for the Subdirectory of extracted archive. PortableGit will be installed to `$JENKINS_HOME\tools\git.exe` and the fully qualified path to the git exe will be `$JENKINS_HOME\tools\git.exe\bin\git.exe`
```

Comment:
Thanks for the pull request!  This file prefers sentence per line formatting so that code review is easier.  I still need to test the instructions before we merge this change.

It would be clearer (at least to me) if the archive that is created were named "git-2.28.0" rather than "git.exe".  Having a directory named "git.exe" would tend to confuse me.  Is there something I'm not understanding which makes it critical that the directory must be named "git.exe" rather than (for example) "git-2.28.0"?
```suggestion
The git client plugin supports installing PortableGit automatically from a zip or tar.gz file.
First follow PortableGit's installation instructions and then repack so that the git executable is at `<zipname>.zip\bin\git.exe`.
Leave Path to Git executable as git.exe and specify `bin\git.exe` for the Subdirectory of extracted archive.
PortableGit will be installed to `$JENKINS_HOME\tools\git.exe` and the fully qualified path to the git exe will be `$JENKINS_HOME\tools\git.exe\bin\git.exe`
```